SQL/4-CreateRetreatOptions.sql

```diff
@@ -0,0 +1,35 @@
+BEGIN TRANSACTION;
+
+CREATE TABLE retreat_options (
+    board_id int,
+    phase text,
+    origin text,
+    retreat_loc text,
+    PRIMARY KEY (board_id, phase, origin, retreat_loc),
+    FOREIGN KEY (board_id, phase, origin) REFERENCES provinces (board_id, phase, province_name),
```

Comment:
Not sure if it matters but can add a foreign key referencing boards (board_id, phase)

Reply:
Nvm you just added that haha